tools/its.py

```diff
@@ -219,20 +219,22 @@ def driver_enter_code(self, driver, plz_impfzentrum):
         driver.get(url)
 
         # Queue Bypass
-        queue_cookie = driver.get_cookie("akavpwr_User_allowed")
-        if queue_cookie:
-            self.log.info("Im Warteraum, Seite neuladen")
+        while True:
+            queue_cookie = driver.get_cookie("akavpwr_User_allowed")
+            if not queue_cookie:
+                break
+            self.log.info("Im Warteraum, Seite neu laden")
             queue_cookie["name"] = "akavpau_User_allowed"
             driver.add_cookie(queue_cookie)
 
             # Seite neu laden
+            time.sleep(5)
```

Comment:
Wozu der Sleep?

Reply:
In meiner Erfahrung dauert es oftmals so eine halbe Minute um den Warteraum zu skippen, egal ob man es in der Zeit ganz oft, oder nur alle 5 Sekunden probiert.